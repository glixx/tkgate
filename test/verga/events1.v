module top;
  event e;

  initial
    begin
	#0;
	-> e;
	-> e;
	#10;
	-> e;
	-> e;
    end

  always @(e)
    $display("@%d: triggered on e",$time);


endmodule
