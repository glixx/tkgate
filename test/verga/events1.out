@0: triggered on e
@10: triggered on e
